iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/MultiplexingClientTests.java

```diff
@@ -918,15 +918,15 @@ private static void testDesiredPropertiesFlow(DeviceClient deviceClient, TwinCli
     private static void testReportedPropertiesFlow(DeviceClient deviceClient, TwinClient twinClientServiceClient, String expectedPropertyKey, String expectedPropertyValue) throws IOException, IotHubException, InterruptedException, TimeoutException, IotHubClientException
     {
         String expectedReportedPropertyValue = expectedPropertyValue + "-reported";
-        TwinCollection reportedProperties = new TwinCollection();
+        TwinCollection reportedProperties = deviceClient.getTwin().getReportedProperties();
```

Comment:
Getting the initial twin in case this identity has been used in a previous test and has an initial twin already